robot_sdk/wrappers/ec_wrapper.py

```
@@ -91,18 +109,43 @@ def set_hand_teach(self, enable: bool) -> Optional[bool]:
+            except Exception as e:
+                print(f"Error moving to pose: {e}")
+                return False, None
+            
+        elif self.planner_type == "moveit":
+            print("Using MoveIt planner for Cartesian movement")
+            try:
+                if not isinstance(pose, list) or len(pose) != 6:
+                    print("Error: Pose must be a list of 6 values [x, y, z, rx, ry, rz]")
+                    return False, None
+                
+                pose = [pose[0]*1000, pose[1]*1000, pose[2]*1000] + pose[3:]  # Convert m to mm for coordinate frame transformations
```

Ruff (0.13.1)

Consider iterable unpacking instead of concatenation

(RUF005)